classes/Pods.php

```diff
@@ -4175,6 +4175,16 @@ private function process_magic_tags( $tag ) {
 			$value = $this->display( $field_name );
 		}
 
+		// Process special magic tags but allow "empty" values for numbers.
+		if (
+			! $value
+			&& ! is_numeric( $value )
+			&& pods_shortcode_allow_evaluate_tags()
+			&& ! $this->fields( $field_name )
+		) {
+			$value = pods_evaluate_tag( implode( ',', $tag ) );
+		}
+
```

Comment:
```suggestion
```

Reply:
This part is required for this PR to work properly. It's based on PR: #5713 which will be closed automatically once this PR is merged.